### DISCUSSION TITLE
Manually change a credentials key for a particular use case

### DISCUSSION THREAD

**Discussion by JamesFH95** [postalserver/postal#1949] [Help with using Postal]
Hey there i would like to manually set a Key for one of my login credentials for a particular use case I want to test. Is this possible?

**Comment by willpower232** [comment]
We've had to do this in the past where a printer refused to accept a password of a certain length, if you create a credential in the UI then you can edit it directly in the database to whatever value/length you need

**Reply by JamesFH95** [reply]
What would the folder path of this DB be?

**Reply by willpower232** [reply]
If you use a GUI to access the databases, you will be able to find the right table, I'm not sure off the top of my head

**Reply by JamesFH95** [reply]
All I've got right now is just WinSCP and I'm logged into the server trying to find the DB file. When you say a GUI do you mean like DB beaver? I can connect this to my postal server and I should be able to see the databases?

**Reply by willpower232** [reply]
the database is not a file, MySQL (and MariaDB) databases should only be edited with a MySQL-compatible client such as DBeaver, HeidiSQL, or even the cli if that is how you work in databases. I would definitely urge caution if you aren't that familiar with editing MySQL databases.

**Reply by JamesFH95** [reply]
Thanks Will I will be careful
